manifests/init.pp

```diff
@@ -171,6 +177,8 @@
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   String[1] $remote_user_environ_name = 'HTTP_REMOTE_USER',
   Integer[0] $worker_count = min(8, $facts['processors']['count']),
+  Boolean $service_enable = true,
+  Boolean $service_ensure = true,
```

Comment:
We can also use running/stopped.
```suggestion
  Stdlib::Ensure::Service $service_ensure = 'running',
```

Reply:
Maybe I am missing a nuance, but it appears to validate that its a boolean for `systemd::unit_file`:

https://github.com/camptocamp/puppet-systemd/blob/master/manifests/unit_file.pp#L60

Reply:
While it would both work, by using the type alias that uses `running` and `stopped` you would avoid the casting in service.pp (around line 40). There also has been a long standing convention that `running` and `stopped` should be used. `ensure => running` reads a lot better than `ensure => true` since it's self-documenting.

Reply:
Sorry, I am confused still as to what the design change ask here is. Are you saying it's better to go with `running` and `stopped` and cast that to a Boolean inside the manifest for the `systemd::unit_file` given it requires a boolean?

Reply:
Ah, I missed that nuance and assumed it accepted running/stopped.

@alexjfisher what do you think is a cleaner API?

Reply:
Seems this is waiting on your input @alexjfisher and I'd like to close this out to do the follow on work.